specutils/tests/test_loaders.py

```diff
@@ -522,14 +522,14 @@ def test_tabular_fits_writer(tmpdir, spectral_axis):
         spectrum.write(tmpfile, format='tabular-fits')
     spectrum.write(tmpfile, format='tabular-fits', overwrite=True)
 
-    cmap = {spectral_axis: ('spectral_axis', wlu[spectral_axis]),
+    cmap = {spectral_axis: ('spectral_axis', 'micron'),
```

Comment:
Why was this changed? It seems to defeat the purpose of the parameterized test.

Reply:
`wlu[spectral_axis]` is simply the original unit `disp` was created with, so for `spec.spectral_axis` the original test did not cover custom unit conversion by the mapping (and therefore did not find the bug of not identifying `energy` as a `physical_type` correctly).
I guess this might be moved to a separate check, but I don't see any problems with testing `spectral_axis` and `flux` units in the same run.